src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -26,42 +23,93 @@ public class StationAcceptanceTest {
     @DisplayName("지하철역을 생성한다.")
     @Test
     void createStation() {
-        // when
-        Map<String, String> params = new HashMap<>();
-        params.put("name", "강남역");
-
-        ExtractableResponse<Response> response =
-                RestAssured.given().log().all()
-                        .body(params)
-                        .contentType(MediaType.APPLICATION_JSON_VALUE)
-                        .when().post("/stations")
-                        .then().log().all()
-                        .extract();
+        // given
+        String gasan = "가산디지털단지역";
 
-        // then
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        // when
+        createStationRequest(gasan);
 
         // then
-        List<String> stationNames =
-                RestAssured.given().log().all()
-                        .when().get("/stations")
-                        .then().log().all()
-                        .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+        assertThat(requestListAndExtractStationInfo("name")).containsAnyOf(gasan);
     }
 
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @Test
+    @DisplayName("지하철역 2개를 생성하고, 목록을 조회한다.")
+    void showStations() {
+        //given
+        String gasan = "가산디지털단지역";
+        String guro = "구로디지털단지역";
+
+        createStationRequest(gasan);
+        createStationRequest(guro);
+
+        // when
+        List<String> stationNames = requestListAndExtractStationInfo("name");
+
+        // then
+        assertThat(stationNames).containsAnyOf(gasan);
+        assertThat(stationNames).containsAnyOf(guro);
```

Comment:
containsAnyOf 메서드는 가변인자를 받고 있으니 assert 구문을 하나로 줄이면 어떨까요?

```suggestion
        assertThat(stationNames).containsAnyOf(gasan, guro);
```

순서까지 검증할 수 있는 containsExactly 메서드를 활용해 보셔도 좋을 것 같아요

Reply:
맞습니다. 더 깔끔한 코드가 된 것 같습니다.

Reply:
맞습니다. 더 깔끔한 코드가 된 것 같습니다.